arch/xtensa/src/esp32s2/Kconfig

```diff
@@ -683,6 +681,14 @@ endmenu # SPI Flash configuration
 menu "SPI RAM Config"
```

Comment:
This menu is out of place.
Please move it outside of the `ESP32-S2 Peripheral Selection` menu.